### ISSUE TITLE
Add Save/load State 

### ISSUE THREAD

**Issue by leafbreakerdev** [weihuoya/citra#808]
We already have per game settings this is great!! Now we need save/load state please

**Comment by Tappy-tech** [comment]
For the 2nd time repeated by me, Weihuoya won't add save states!!

**Comment by AndresNorm** [comment]
He Can't.

**Comment by scriptprojectsdev** [comment]
@Tappy-tech @AndresNorm there's a lot of issues opened about save states without an answer. Just saying he won't or can't will keep people asking the same question over and over again. Can you link to an explanation as to why? Or explain it here so when people search through the issues they'll find out why this feature won't or can't be added.

**Comment by AndresNorm** [comment]
> @Tappy-tech @AndresNorm there's a lot of issues opened about save states without an answer. Just saying he won't or can't will keep people asking the same question over and over again. Can you link to an explanation as to why? Or explain it here so when people search through the issues they'll find out why this feature won't or can't be added.

https://github.com/weihuoya/citra/pull/738

I think this is the one, Inform me if im Wrong Though

**Comment by Tappy-tech** [comment]
#738 seems to be more about how Citra MMJ doesn't have it's modifications documented in readme.md from the original Citra. 

I'm sure that the statement regarding save states (Mine) was taken from his Weibo, where he answers that he won't add them.

**Comment by AndresNorm** [comment]
> #738 seems to be more about how Citra MMJ doesn't have it's modifications documented in readme.md from the original Citra. 
> 
> I'm sure that the statement regarding save states (Mine) was taken from his Weibo, where he answers that he won't add them.

Yep and Weihouya didn't provide one to its Modification which is a Violation of License
